### PR TITLE
[gguf] document new types

### PR DESCRIPTION
**Pull request by mishig25**
Adding new types from https://github.com/huggingface/huggingface.js/pull/729

cc: @CISC

### CODE REVIEW

File: docs/hub/gguf.md [hunk 1]
```diff
@@ -77,13 +83,14 @@ Find more information [here](https://github.com/huggingface/huggingface.js/tree/
...
 | IQ2_S | [HF](https://huggingface.co/CISCai/OpenCodeInterpreter-DS-6.7B-SOTA-GGUF/blob/main/README.md?code=true#L59-L70) | 2-bit quantization (`q`). Super-blocks with 256 weights. Weight `w` is obtained using `super_block_scale` & `importance matrix`, resulting in 2.5 bits-per-weight. |
 | IQ2_XS | [HF](https://huggingface.co/CISCai/OpenCodeInterpreter-DS-6.7B-SOTA-GGUF/blob/main/README.md?code=true#L59-L70) | 2-bit quantization (`q`). Super-blocks with 256 weights. Weight `w` is obtained using `super_block_scale` & `importance matrix`, resulting in 2.31 bits-per-weight. |
 | IQ1_S | [HF](https://huggingface.co/CISCai/OpenCodeInterpreter-DS-6.7B-SOTA-GGUF/blob/main/README.md?code=true#L59-L70) | 1-bit quantization (`q`). Super-blocks with 256 weights. Weight `w` is obtained using `super_block_scale` & `importance matrix`, resulting in 1.56 bits-per-weight. |
-| IQ4_NL | [GH](https://github.com/ggerganov/llama.cpp/pull/5590) | 4-bit quantization (`q`). Super-blocks with 256 weights. Weight `w` is obtained using `super_block_scale` & `importance matrix`. |
```
Comment (osanseviero) [inline docs/hub/gguf.md:87]: Why is this being removed?

Comment (mishig25) [reply docs/hub/gguf.md:87]: I re-orderd that line https://github.com/huggingface/hub-docs/blob/cb4f84a1d29d197c46dde34bd27bc2d3eb36c5d9/docs/hub/gguf.md#L86-L87

to make it close to other `IQ4-xyz`

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
(how do you generate it btw? just code editor ninja skills?)

**Comment by mishig25** [comment]
Atm, the process is manual. There is no automatic script for two reasons:
1. the order shown on this table is different from there order shown on gguf.js
2. The text written in gguf.js is plain text. In the docs, I try to highlight the codes with markdown when appropriate

**Pull request merged**